### PR TITLE
Implement sample version of Comment feature.

### PR DESCRIPTION
**Pull request by chenyuncai**
Implement the comment feature.

1. add comment-run element.
2. add document attr config for comment feature.
3. add content-type and relationships item.
4. add a comment builder demo.
5. change the packer prettier param to string as sometime you can pass two blanks to xml output.

### CODE REVIEW

**Comment by chenyuncai** [comment]
@dolanmiu Hi, there is PR about word comment feature.

File: src/file/paragraph/run/comment-run.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+interface ICommentOptions {
+    readonly id: string;
+    readonly initials?: string;
+    readonly author?: string;
+    readonly date?: string;
+}
+
+class CommentAttributes extends XmlAttributeComponent<ICommentOptions> {
+    protected readonly xmlKeys = { id: "w:id", initias: "w:initials", author: "w:author", date: "w:date" };
```
Comment (VictorBaron) [inline src/file/paragraph/run/comment-run.ts:13]: typo here: `initias` --> `initials`

File: src/import-dotx/import-dotx.ts [hunk 2]
```diff
@@ -37,11 +38,12 @@ interface IRelationshipFileInfo {
...
-    readonly headers: IDocumentHeader[];
-    readonly footers: IDocumentFooter[];
-    readonly styles: string;
-    readonly titlePageIsDefined: boolean;
-    readonly media: Media;
+    readonly headers?: IDocumentHeader[];
+    readonly footers?: IDocumentFooter[];
+    readonly styles?: string;
+    readonly titlePageIsDefined?: boolean;
+    readonly media?: Media;
```
Comment (dolanmiu) [inline src/import-dotx/import-dotx.ts:45]: Why are these now optional?

File: src/export/packer/next-compiler.ts [hunk 3]
```diff
@@ -64,7 +65,7 @@ export class Compiler {
         return zip;
     }
 
-    private xmlifyFile(file: File, prettify?: boolean): IXmlifyedFileMapping {
+    private xmlifyFile(file: File, prettify?: boolean | string): IXmlifyedFileMapping {
```
Comment (dolanmiu) [inline src/export/packer/next-compiler.ts:68]: It is now a boolean and a string, what could the string be used for?

Comment (dolanmiu) [reply src/export/packer/next-compiler.ts:68]: It seems like it's used like so:

```ts
Packer.toBuffer(doc, '  ').then((buffer) => {
  fs.writeFileSync("document-comments.docx", buffer);
});
```

I would prefer it if it is passed as an option in `File` instead

Comment (chenyuncai) [reply src/export/packer/next-compiler.ts]: Change the normal string to a enum type to protect the xml output content.

**Pull request closed without merging**